Projects/Team3/Smart_House/Smart_House/House.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Smart_House
+{
+    class House
```

Comment:
As a recommendation for the future, I would recommend separate logic looking at the single responsibility principle. 
For example, House can have sensors, conditioners and etc, but I would move controlling methods to some kind of controller, like an IoT controller for managing sensors, heaters, and etch.

Reply:
Thank  you for review  and  idea)

Reply:
Thanks) We will try to implement your ideas in the future)))